### PR TITLE
Proton Whitelisting

### PR DESCRIPTION
**Pull request by daddeltrotter**
Hello,
I added the games I tested in the past two months.
In the past days, I changed my testing approach and did it first with the latest stable version, 
hence majority of test were done on the latest development verion (which turned out to be quite stable).
Thanks for reviewing. If I missed something or did something wrong, please let me know.
Best regards,
Daddeltrotter

### CODE REVIEW

File: steam-tweaks.yaml [hunk 1]
```diff
@@ -9,9 +9,18 @@
 "435790": # 10 Second Ninja X
   compat_tool: proton_411
 
+"238930": # 7 Grand Steps, Step 1: What Ancients Begat
```
Comment (alkazar) [inline steam-tweaks.yaml:12]: The game name appears to be "7 Grand Steps: What Ancients Begat" according to Steam.

File: steam-tweaks.yaml [hunk 2]
```diff
@@ -82,15 +124,39 @@
...
+  compat_tool: proton_513
+
+"403290": # Burly Men at Sea
+  compat_tool: proton_513
+
 "24740": # Burnout Paradise: The Ultimate Box
   compat_tool: proton_411
 
+"598810": # Carcassonne: The Official Board Game
+  compat_tool: proton_51
```
Comment (alkazar) [inline steam-tweaks.yaml:152]: Is this supposed to be `proton_513`? I don't think there is a `proton_51`.

File: steam-tweaks.yaml [hunk 3]
```diff
@@ -100,16 +166,52 @@
...
   compat_tool: proton_411
 
+"26900": # Crayon Physics Deluxe
+  compat_tool: proton_513
+
 "71230": # Crazy Taxi
   compat_tool: proton_411
   compat_config: d9vk
 
+“829590”: # Crying Suns
```
Comment (alkazar) [inline steam-tweaks.yaml:200]: Looks like the ID should be "873940".

Comment (alkazar) [inline steam-tweaks.yaml:200]: The quote characters are not correct.

File: steam-tweaks.yaml [hunk 4]
```diff
@@ -100,16 +166,52 @@
...
+  compat_tool: proton_5
+
+“829590”: # Cryofall (Beta Proton Release of devs)
```
Comment (alkazar) [inline steam-tweaks.yaml:203]: quote characters are not correct, but let's wait until this is working with the full release and not just the beta.

File: steam-tweaks.yaml [hunk 5]
```diff
@@ -100,16 +166,52 @@
...
+  compat_tool: proton_513
+
+“928840”: # Cybarian: The Time Travelling Warrior
```
Comment (alkazar) [inline steam-tweaks.yaml:206]: quotes

File: steam-tweaks.yaml [hunk 6]
```diff
@@ -100,16 +166,52 @@
...
+  compat_tool: proton_5
+
+“405500”: # Dangerous Golf
```
Comment (alkazar) [inline steam-tweaks.yaml:209]: quotes

File: steam-tweaks.yaml [hunk 7]
```diff
@@ -382,6 +487,9 @@
 "589360": # Ni no Kuni II: Revenant Kingdom
   compat_tool: proton_411
 
+"94400": # Nidhogg
+  compat_tool: proton_51
```
Comment (alkazar) [inline steam-tweaks.yaml:491]: `proton_513`?

File: steam-tweaks.yaml [hunk 8]
```diff
@@ -516,8 +624,12 @@
 "574980": # Them's Fightin' Herds
   compat_tool: proton_42
 
+"108200": # Ticket to Ride
+  compat_tool: proton_51
```
Comment (alkazar) [inline steam-tweaks.yaml:628]: `proton_513`?

**Review by alkazar — CHANGES_REQUESTED** [review]

File: steam-tweaks.yaml (hunk 1 above)
Comment (daddeltrotter) [reply steam-tweaks.yaml:12]: Mmmh..right. In my Proton list it shows with "step1". Strange...Chenged it

File: steam-tweaks.yaml (hunk 2 above)
Comment (daddeltrotter) [reply steam-tweaks.yaml:152]: 5

File: steam-tweaks.yaml (hunk 3 above)
Comment (daddeltrotter) [reply steam-tweaks.yaml:200]: Right...Copy&paste :(

Comment (daddeltrotter) [reply steam-tweaks.yaml:200]: Damn...

File: steam-tweaks.yaml (hunk 4 above)
Comment (daddeltrotter) [reply steam-tweaks.yaml:203]: I commented the game and keep watching it.

File: steam-tweaks.yaml (hunk 5 above)
Comment (daddeltrotter) [reply steam-tweaks.yaml:206]: Corrected

File: steam-tweaks.yaml (hunk 6 above)
Comment (daddeltrotter) [reply steam-tweaks.yaml:209]: Corrected

File: steam-tweaks.yaml (hunk 7 above)
Comment (daddeltrotter) [reply steam-tweaks.yaml:491]: 5

File: steam-tweaks.yaml (hunk 8 above)
Comment (daddeltrotter) [reply steam-tweaks.yaml:628]: 5

**Comment by daddeltrotter** [comment]
Made all the changes. Sorry for my inattentions...

File: steam-tweaks.yaml [hunk 9]
```diff
@@ -100,16 +166,52 @@
...
+  compat_tool: proton_513
+
 "71230": # Crazy Taxi
   compat_tool: proton_411
   compat_config: d9vk
 
+"873940": # Crying Suns
+  compat_tool: proton_5
+
+# "829590": # Cryofall (Beta Proton Release of devs)
```
Comment (alkazar) [inline steam-tweaks.yaml:203]: Please remove entirely. We can add it again when it is ready.

**Review by alkazar — CHANGES_REQUESTED** [review]
Thanks for the fixes! Just one more small thing.

File: steam-tweaks.yaml (hunk 9 above)
Comment (daddeltrotter) [reply steam-tweaks.yaml]: Done

**Comment by alkazar** [comment]
Nice work, thanks!

**Pull request merged**